articles/azure-sql/database/database-export.md

```diff
@@ -98,6 +98,9 @@ while ($exportStatus.Status -eq "InProgress")
 [Console]::WriteLine("")
 $exportStatus
 ```
+## Cancel the export request
```

Comment:
## Cancel the export request
Use the [Database Operations - Cancel API] (https://docs.microsoft.com/en-us/rest/api/sql/databaseoperations/cancel)
or the Powershell [Stop-AzSqlDatabaseActivity command] (https://docs.microsoft.com/en-us/powershell/module/az.sql/Stop-AzSqlDatabaseActivity?view=azps-5.5.0)

Example:-
Stop-AzSqlDatabaseActivity -ResourceGroupName $ResourceGroupName -ServerName $ServerName -DatabaseName $DatabaseName -OperationId $Operation.OperationId